fairlearn/preprocessing/_informationfilter.py

```diff
@@ -0,0 +1,120 @@
+import numpy as np
+import pandas as pd
+from sklearn.base import BaseEstimator, TransformerMixin
+from sklearn.utils import check_array
+from sklearn.utils.validation import check_is_fitted
+from sklearn.preprocessing import StandardScaler
+
+
+def _scalar_projection(vec, unto):
+    return vec.dot(unto) / unto.dot(unto)
+
+
+def _vector_projection(vec, unto):
+    return _scalar_projection(vec, unto) * unto
+
+
+class InformationFilter(BaseEstimator, TransformerMixin):
+    r"""
+    The `InformationFilter` filters out sensitive correlations in a dataset.
+
+    The `InformationFilter` uses a variant of the gram smidt process
```

Comment:
Did you mean Gram-Schmidt?

Reply:
also occurs in the code itself

Reply:
It seems my dyslexia is making a come-back. Will fix.